pipelines/controller/src/main/java/com/google/fhir/analytics/DwhFilesManager.java

```diff
@@ -65,8 +67,12 @@ public class DwhFilesManager {
   private boolean isPurgeJobRunning;
 
   private String dwhRootPrefix;
+
   private int numOfDwhSnapshotsToRetain;
 
+  private static final Pattern WINDOWS_SCHEME_PATTERN =
+      Pattern.compile("(?<scheme>[a-zA-Z]):\\\\(?<path>.*)");
```

Comment:
I am a little bit confused about these double `\`s; I thought `C:\test\file.txt` is a valid path in Windows.

Reply:
As discussed, because of the way the literal String is handled in Java (backslash is a escape charater), we need 2 backslashes for defining a backslash. Also the `Pattern` class treats backslash also as escape character, hence we need 2 more slashes for that. However, in the latest commit this has been removed.